### ISSUE TITLE
Ad support for Zephyr 3.0

### ISSUE THREAD

**Issue by mdkf** [platformio/zephyr#15]
A new version of Zephyr was released yesterday: https://github.com/zephyrproject-rtos/zephyr/releases/tag/zephyr-v3.0.0

**Comment by cpavot** [comment]
Hello guys, do you plan to release Zephyr 3.0 support in the coming weeks ?

**Comment by larmicfre** [comment]
This needs to happen. Let us have 3rd version Zephyr! @ivankravets

**Comment by barchard** [comment]
The latest now is 3.1.0,

https://github.com/zephyrproject-rtos/zephyr/releases/tag/zephyr-v3.1.0

**Comment by cpavot** [comment]
Hello @ivankravets, can you please share if you plan to integrate the latest version of Zephyr ?

**Comment by danielkucera** [comment]
Hi,
@ivankravets , @valeros , can you please at least announce what do you plan with zephyr future?
Continue support or deprecate?

**Comment by JamesCCorralTech** [comment]
Zephyr 3.2.0 is out now as well. I would love to use PIO but the version included with PIO is missing many fixes and features now.

**Comment by ivankravets** [comment]
What are your boards/hardware?

**Comment by barchard** [comment]
I've been waiting for this too. We use NRF52833

**Comment by JamesCCorralTech** [comment]
bl654_dvk (NRF52840)

**Comment by antoniomolram** [comment]
> 

I would like also an upgrade, due to Arduino Nicla Sense Me from Bosch Sensortec support inside of Zephyr 3.2 (or maybe older?)

**Comment by larmicfre** [comment]
STM32; F1, F2 and F4 series are the most common.

**Comment by antoniomolram** [comment]
@ivankravets , any update on this? Thank you for your work!

**Comment by beriberikix** [comment]
FYI https://github.com/zephyrproject-rtos/zephyr/pull/53303

**Comment by danielkucera** [comment]
Sad but true :disappointed:

**Comment by zpm1066** [comment]
Yes, disappointing that PlatformIO can no longer be recommended as a one-stop IDE.

It would have been great to have PlatformIO supporting the latest Zephyr running on nRF52, nRF53, STM32, RP2040, and ESP32.
